javascript-view-swapping/main.js

```diff
@@ -0,0 +1,34 @@
+var tabContain = document.querySelector('.tab-container');
+var tabElements = document.querySelectorAll('.tab');
+var viewElements = document.querySelectorAll('.view');
+
+function tabList(event) {
+  if (event.target.matches('.tab')) {
+    for (var i = 0; i < tabElements.length; i++) {
+      if (event.target === tabElements[i]) {
+        tabElements[i].className = 'tab active';
+      } else tabElements[i].className = 'tab';
+    }
+
+    if (event.target.matches('.tab')) {
+      var eventT = event.target.getAttribute('data-view');
+      for (i = 0; i < viewElements.length; i++) {
+        if (eventT === 'html') {
```

Comment:
while this works, I would think about what to do if you had 10 views or more? instead of setting a conditional statement for each case, you just want to check the attribute of the element which was clicked on, and switch that one to show and hide the rest.